packages/flutter_tools/test/general.shard/android/android_workflow_test.dart

```diff
@@ -531,7 +529,7 @@ Review licenses that have not been accepted (y/N)?
 
     final ValidationMessage cmdlineMessage = validationResult.messages.last;
     expect(cmdlineMessage.type, ValidationMessageType.error);
-    expect(cmdlineMessage.message, errorMessage);
+    expect(cmdlineMessage.message, startsWith('cmdline-tools component is missing.\n'));
```

Comment:
Drive-by comment, but should we also check that

`https://developer.android.com/studio#command-line-tools-only` and/or `https://developer.android.com/studio/command-line` is in the message?
And perhaps even that `Run path/to/sdkmanager` is _not_ in the message.

The first phrase of the old message only different by a period at the end, and this test doesn't test that 
`Run `path/to/sdkmanager --install "cmdline-tools;latest"` is actually gone, which is the part we're trying to avoid here?